src/index.js

```diff
@@ -12,7 +12,7 @@ export Column from './Grid/Column';
 export Container from './Grid/Container';
 export Dropdown from './Dropdown';
 export Field from './Form/Field';
-export FieldSet from './Form/FieldSet';
+export Fieldset from './Form/Fieldset';
```

Comment:
Let's rename the file too - would cause issues buliding on OSs like Linux that care about path capitalization